mgmt_cli_test.py

```diff
@@ -199,17 +200,27 @@ def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspac
             self.restore_backup_from_backup_task(mgr_cluster=mgr_cluster, backup_task=backup_task,
                                                  keyspace_and_table_list=per_keyspace_tables_dict)
 
-    def restore_schema_with_task(self, mgr_cluster, backup_task, timeout):
+    def restore_schema_with_task(self, mgr_cluster, backup_task, timeout, location_list=None):
         snapshot_tag = backup_task.get_snapshot_tag()
-        restore_task = mgr_cluster.create_restore_task(restore_schema=True, location_list=self.locations,
+        self.restore_schema_with_task_from_snapshot_tag(mgr_cluster=mgr_cluster, snapshot_tag=snapshot_tag,
+                                                        timeout=timeout, location_list=location_list)
+
+    def restore_schema_with_task_from_snapshot_tag(self, mgr_cluster, snapshot_tag, timeout, location_list=None):
```

Comment:
From the name it looks like it restores only the schema and not the data.

Reply:
Because it does...
the restore-schema option only restores the schema, without the data.